provision/acc_provision/flavors.yaml

```diff
@@ -212,6 +212,132 @@ flavors:
     status: null
     hidden: false
     order: 0
```

Comment:
this needs to be updated to 1 (and all the other order numbers need to be incremented by 1).

Reply:
noted